config/v1/types_infrastructure.go

```diff
@@ -614,6 +626,14 @@ type OpenStackPlatformStatus struct {
 	// datacenter DNS, a DNS service is hosted as a static pod to serve those hostnames
 	// to the nodes in the cluster.
 	NodeDNSIP string `json:"nodeDNSIP,omitempty"`
+
+	// loadBalancerType defines if a internal or external load balancer
+	// is to be used for the openshift api and ingress endpoints.
+	// When set to Internal the static pods in charge of API and Ingress traffic load-balancing
+	// defined in machine config operator will be deployed.
+	// When set to External these static pods will not be deployed.
```

Comment:
I'd add something about the default case. Something like: When left empty, the choice is left to the default, which at the time of writing is `Internal`.

Reply:
Good idea. Will do once my other comment is answered and when I submit a change.